examples/basic/config.yml

```diff
@@ -7,6 +7,7 @@ repository: https://github.com/sergiodxa/personal-site/
 email: hello@sergiodxa.com
 patreon: sergiodxa
 paypal: sergiodxa
+analytics: UA-138242722-2
```

Comment:
Is this a real analytics ID? If so could you replace it with a fake one?

Reply:
I do not know if it is a real code, change all the numbers at random, at least it does not belong to me, but yes, we can change it for a more generic one.

Reply:
Try `UA-XXXXXXXXX-2` which is totally fake